.github/workflows/develop-ci-cd.yml

```
@@ -0,0 +1,147 @@
+      - name: Login to ECR
+        if: ${{ contains(env.services_to_build, matrix.service.folder) }}
+        uses: aws-actions/amazon-ecr-login@v2
+
+      # 8. Docker 이미지 빌드 및 ECR 푸시
+      - name: Build, Tag and Push Docker Image to ECR
+        if: ${{ contains(env.services_to_build, matrix.service.folder) }}
+        run: |
+          cd ${{ matrix.service.folder }}
+          docker build -t ${{ matrix.service.name }} .
+          docker tag ${{ matrix.service.name }}:latest ${{ secrets.ECR_REGISTRY }}/${{ matrix.service.name }}:latest
+          docker push ${{ secrets.ECR_REGISTRY }}/${{ matrix.service.name }}:latest      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)